### DISCUSSION TITLE
`Audacity` not recognising `ffmpeg-shared`

### DISCUSSION THREAD

**Discussion by southisup** [ScoopInstaller/Extras#11098] [Q&A] [answered]
As subject. I'm almost certain it used to work, but now only `avformat-60.dll` is present and Audacity silently ignores that when pointed at it, simply continuing to report library not found. Any ideas?

```
PS C:\Users\user> scoop status; scoop list audacity; scoop list ffmpeg
Scoop is up to date.
Everything is ok!
Installed apps matching 'audacity':

Installed apps matching 'ffmpeg':
Name          Version Source Updated             Info
----          ------- ------ -------             ----
audacity      3.2.5   extras 2023-03-03 18:57:13
ffmpeg-shared 6.0     main   2023-03-03 18:26:41


PS C:\Users\user>
```

**Comment by HUMORCE** [comment]
> - Audacity 3.1 and later supports avformat-55.dll, avformat-57.dll and avformat-58.dll. Audacity 3.2 and later supports also avformat-59.dll. You can check which dll is in which FFmpeg release [here](https://ffmpeg.org/download.html#releases). 
> - Make sure you download full FFmpeg copies, not just the avformat-*.dll's individually. Further, make sure to download or build the shared versions as only those contain .dll's. 
> - Different versions of FFmpeg may have different codecs enabled in them. In particular, AMR (narrowband) is not featured in the recommended installer.
> 
> Source: https://support.audacityteam.org/basics/installing-ffmpeg#other-ffmpeg-builds


**Audacity 3.2 and later supports also avformat-59.dll.**

**Reply by southisup** [reply]
So that means scoop main bucket's `ffmpeg-shared` is incompatibly ahead of extra bucket's `audacity`? Is there a documented way in scoop to mitigate this?

**Reply by HUMORCE** [reply, marked as answer]
Install older version of `ffmpeg-shared`: `scoop install ffmpeg-shared@5.1.2`

OR create a manifest for https://lame.buanzo.org/ffmpeg.php . if you do so, I recommend that you also submit it to [versions bucket](https://github.com/ScoopInstaller/Versions).

**Reply by southisup** [reply]
Thanks. the `@5.1.2` option works for me, although not until I first uninstalled the current version.
